app/locales/languages.d.ts

```diff
@@ -2,10 +2,15 @@ import {
   LOCALE_LIST as localList,
   setUserLocaleOverride as setOverride,
   supportedDeviceLanguageOrEnglish as supportedDevice,
+  useLanguageDirection as languageDirection,
 } from './languages';
 
 export const LOCALE_LIST: Record<unknown, unknown> = localList;
 export const setUserLocaleOverride: (
   locale: string,
 ) => Promise<void> = setOverride;
 export const supportedDeviceLanguageOrEnglish: () => string = supportedDevice;
+export const useLanguageDirection: () =>
```

Comment:
ideally we update languages.js to ts as well at some point,

i initially thought to inline this function to the component as well, since it was a bit odd to have it pulled out imo, and wed get the type definitions as well, but it turns out the test suite (and the whole app) is depending on some setup that runs when this is imported.

It might makes sense to pull that initialization logic into a more obvious initialization place. This would be less prone to be forgotten about and we'd be able to mock it tests. Probably not a priority, but wanted to mention it as a thing.